src/model/constants.rs

```diff
@@ -1,14 +1,21 @@
-
+// Model constants
 pub static MULTIPLIER: f64 = 45.0;
 pub static SIGMA: f64 = (25.0 / 5.0) * MULTIPLIER;
 pub static MU: f64 = 15.0 * MULTIPLIER;
 pub static TAU: f64 = SIGMA / 100.0;
 pub static BETA: f64 = SIGMA / 2.0;
 pub static KAPPA: f64 = 0.0001;
 pub static DECAY_DAYS: u64 = 115;
-pub static DECAY_MINIMUM: f64 = 825.0;
+pub static DECAY_MINIMUM: f64 = MULTIPLIER * 18.0;
```

Comment:
```suggestion
pub static DECAY_MINIMUM: f64 = 18.0 * MULTIPLIER;
```
This probably seems kind of pedantic, but I realized that we should be thinking of MU and DECAY_MINIMUM as similar to each other (like in general we should have MU less than DECAY_MINIMUM so most people won't decay from their starting rating), so it's good to make them look the same to remember that.